src/registerinitializationwidget.cpp

```diff
@@ -27,23 +27,38 @@ void RegisterSelectionComboBox::showPopup() {
   const auto &procisa = ProcessorRegistry::getAvailableProcessors()
                             .at(m_parent->m_currentID)
                             ->isaInfo();
-  const auto regInfo = procisa.isa->regInfo().value();
+  const auto isa = procisa.isa;
```

Comment:
nit: `const auto*`? (if we had clang-tidy checks enabled, it'd probably complain that you're auto'ing a pointer here).